src/PrestaShopBundle/Twig/Extension/GridExtension.php

```diff
@@ -100,7 +100,7 @@ public function renderColumnContent(array $record, array $column, array $grid)
             $template = $this->getTemplatePath(
                 $column,
                 $grid,
-                self::BASE_COLUMN_CONTENT_TEMPLATE_PATH
+                $column['options']['content_template_path'] ?? self::BASE_COLUMN_CONTENT_TEMPLATE_PATH
```

Comment:
```suggestion
                $column['options']['content_template_path']
```

I think we should deprecate `self::BASE_COLUMN_CONTENT_TEMPLATE_PATH`.

Reply:
I don't see any reason to add `BASE_COLUMN_CONTENT_TEMPLATE_PATH` in AbstractColumn having a duplicate value from the GridExtension.
We should only have one value, otherwise, templates should not be the same.
I agree with @Progi1984 if the value is the same we can deprecate one in favor of the other.

Reply:
Ok thanks @PierreRambaud  @Progi1984